src/Illuminate/Validation/Concerns/ValidatesAttributes.php

```diff
@@ -1543,6 +1543,26 @@ public function validateUrl($attribute, $value)
         return preg_match($pattern, $value) > 0;
     }
 
+    /**
+     * Validate that an attribute is a valid UUID.
+     *
+     * @param  string $attribute
+     * @param  mixed  $value
+     * @return bool
+     */
+    public function validateUuid($attribute, $value)
+    {
+        if (! is_string($value)) {
+            return false;
+        }
+
+        if ($value === '00000000-0000-0000-0000-000000000000') {
```

Comment:
🤔

Reply:
Following `ramsey/uuid` validation, this is a nil uuid, which is considered valid. We can choose to accept nil uuids or not. What do you think?

Reply:
I think he means that you just hardcoded a case. Make your regex include that case.

Reply:
Gotcha!

I followed the `ramsey/uuid` validation workflow. They check it agains `00000000-0000-0000-0000-000000000000` before the regex so I did the same.